### DISCUSSION TITLE
[Question]:  does gptPlugins ENDPOINT support custom LLM endpoint ?

### DISCUSSION THREAD

**Discussion by elvis-cai** [danny-avila/LibreChat#3544] [Q&A]
### What is your question?

use LiteLLM with PLUGINS in Librechat

### More Details

I an using gptPlugins with openAI, it works fine, when I change to Litellm, PLUGINS UI section is gone, not sure if it's supported or anything I miss here

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
I'm not sure what you're changing in your configuration that causes the endpoint to disappear, and what you have configured for OpenAI that works, but in general, the plugins endpoint is not expected to work well with a reverse proxy.

There is however something in the works that will natively allow use of tools with custom endpoints. Stay tuned!

**Reply by Phlasse** [reply]
hi @danny-avila I am also currently looking into routing the gptplugins through litellm (mainly because it allows me to trace with langfuse).

any idea on how that can be achieved? or when the thing you are working on might get released? looking forward!

**Reply by danny-avila** [reply]
No need for LiteLLM, “Agents” are the answer to this

> #3607
